netci.groovy

```diff
@@ -137,6 +137,11 @@ branchList.each { branchName ->
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
 
+            // Unix runs take more than 2 hours to run, so we set the timeout to be longer.
```

Comment:
that is correct @mellinoe. Windows runs use Full MSBuild which supports /m and make the build finish fast, but since we have moved Unix to use .Net Core MSBuild everything is serial and takes more than 10 times longer.